### DISCUSSION TITLE
Cosmos UI v1.0 Release Date

### DISCUSSION THREAD

**Discussion by carsonschildt** [TetraOS/.github#8] [Announcements]
Hello There 👋!

As progress is made in the development of TetraOS, so does the UI aspect of it keeps progressing. Over the last couple of weeks the design team has been working hard on constructing and perfecting TetraOS' UI (Cosmos UI). Excitingly, our team has announced that in the next 1-3 weeks, version 1.0 of Cosmos UI will be at it's final design -- so no more significant changes will be made. We are very happy to announce this news, and are hoping you are too.

**Comment by kaleb-bravo** [comment]
#9 Cosmos UI v1.0 is now available!
